scripts/stretch.docker

```diff
@@ -1,9 +1,6 @@
-FROM golang:1.11.0 as builder
-ENV DEP_VERSION 0.5.0
-RUN curl -fsSL -o /usr/local/bin/dep https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 && chmod +x /usr/local/bin/dep
+FROM golang:1.13.8 as builder
 WORKDIR /go/src/github.com/influxdata/telegraf
-COPY Gopkg.toml Gopkg.lock ./
-RUN dep ensure -vendor-only
+
```

Comment:
I wonder why we had the dependency downloading as a separate step.  Perhaps this helps with layer caching when you rebuild the image?

Reply:
I assumed it was required with dep. I guess the change is that I'm no longer venturing it, so there's no ability to reuse the vendored step. We could bring this back with a call to `RUN go mod vendor` right before.

Reply:
I think the intermediate builds get removed anyway, so I'm not sure this would have any impact on release size.